controllers/about/our-people/views/profiles.njk

```diff
@@ -50,7 +50,7 @@
                     {# @TODO: Should we share this component? #}
                     <nav class="nested-menu">
                         <h2 class="nested-menu__section t4">
-                            <a class="nested-menu__section-link" href="./">
+                            <a class="nested-menu__section-link" href="../../../..">
```

Comment:
This is a bit of auto-refactoring weirdness. Not going to merge this branch so can ignore for now.